conf/vertebrates/mlss_conf.xml

```diff
@@ -98,34 +103,206 @@
   <multiple_alignments>
 
     <!-- Primates -->
-    <multiple_alignment method="EPO+EPO_EXTENDED">
+    <multiple_alignment method="EPO">
+      <species_set name="primates">
+        <genome id="60"/>
+        <genome id="150"/>
+        <genome id="153"/>
+        <genome id="198"/>
+        <genome id="199"/>
+        <genome id="206"/>
+        <genome id="209"/>
+        <genome id="210"/>
+        <genome id="221"/>
+        <genome id="222"/>
+        <genome id="225"/>
+        <genome id="317"/>
+        <genome id="361"/>
+      </species_set>
+    </multiple_alignment>
+    <multiple_alignment method="EPO_EXTENDED">
```

Comment:
We don't `gerp` here? (Just checking)

Reply:
It wasn't in the original file, so I guess we use the GERP from mammals instead (or maybe it was a bug? @muffato @carlacummins).

Reply:
Correct, because (I was told) the genomes are too similar and GERP finds too many things. There is the mammals-wise GERP instead, indeed.